lib/modules/manager/cargo/locked-version.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { DirectoryResult, dir } from 'tmp-promise';
+import { join } from 'upath';
+import { Fixtures } from '../../../../test/fixtures';
+import { fs } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
+import { extractLockFileVersions, parseLockFile } from './locked-version';
+
+jest.mock('../../../util/fs');
+
+function mockReadLocalFile(files: Record<string, string | null>) {
+  fs.readLocalFile.mockImplementation((file): Promise<any> => {
+    let content: string | null = '';
```

Comment:
why default value is empty string instead of null?

Reply:
Changed in 7a21fca0b8037590517802070b005d335e333429.